src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { Filter } from '../../services/enums';
+import { TodosContext } from '../../TodosContext';
+import { Todo } from '../../services/types';
+
+function countActiveTodos(todos: Todo[]): number {
+  return todos.filter(todo => {
+    return !todo.completed;
+  }).length;
+}
+
+export const TodoFooter: React.FC = () => {
+  const {
+    todos,
+    handleClearAllCompleted,
+    filterBy,
+    setFilterBy,
+    isTodosHasCompleted,
+  } = useContext(TodosContext);
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${countActiveTodos(todos)} `}
+        items left
```

Comment:
```suggestion
        {' '}items left
```

Reply:
i solved this in such way because of a linter